searx/static/themes/simple/src/less/style.less

```diff
@@ -9,7 +9,7 @@
 @import "../../__common__/less/new_issue.less";
 @import "../../__common__/less/stats.less";
 
-@import "normalize.less";
+@import (less) "../../node_modules/normalize.css/normalize.css";
```

Comment:
* `(css)` doesn't work because: _This means the import statement will be left as it is._ , it should be inlined...
* `(inline)` should work (I'm missed this option).